UPDATING.md

```diff
@@ -100,6 +100,10 @@ serve as a DagBag cache burst time.
 
 The `default_queue` configuration option has been moved from `[celery]` section to `[operators]` section to allow for re-use between different executors.
 
+### Update `parse_s3_url` in airflow.providers.amazon.aws.hooks.s3.S3Hook
+
+The `parse_s3_url` method in `airflow.providers.amazon.aws.hooks.s3.S3Hook` will now not remove the trailing slash by default. This will allow user to more flexibly implement their preferred prefix-level or subdirectory-level s3 operations.
+
```

Comment:
The change of this PR effects only Amazon provider so the note should be in the provider change log https://github.com/apache/airflow/blob/master/airflow/providers/amazon/CHANGELOG.rst

If this change is not backward compatible probably we need to bump major version in the [provider yaml](https://github.com/apache/airflow/blob/master/airflow/providers/amazon/provider.yaml#L24) @potiuk this means 2.0.0 correct?

Reply:
Correct

Reply:
@eladkal @potiuk Hope I have resolved your request

Reply:
Please add also 2.0.0 in the versions section of the provider yaml